packages/ocamline/ocamline.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+authors: "Chris Nevers <christophernevers96@gmail.com>"
+maintainer: "Chris Nevers <christophernevers96@gmail.com>"
+homepage: "https://github.com/chrisnevers/ocamline"
+bug-reports: "https://github.com/chrisnevers/ocamline/issues"
+dev-repo: "git://github.com/chrisnevers/ocamline.git"
+synopsis: "Command line interface for user input"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.9"}
```
Dune should not be tagged as a build dependency (see #14266), moreover, your `dune-project` file says this package requires at least dune 1.9